migrate_pim/upgrade_minor_version.rst

```diff
@@ -1,7 +1,13 @@
-How to upgrade to a minor version?
-==================================
+How to upgrade to a minor version? (PIM version < 4.x)
+======================================================
 
-We release a minor version 3 times a year:
+
+.. note::
+
+    There is no longer minor version since 4.0.
```

Comment:
```suggestion
    Since version 5.0, we have stopped releasing minor versions. Instead, we release more frequent, smaller patches. Instructions for installing patches can be found here: :doc:`/migrate_pim/apply_patch/index`
```